qt/python/mantidqt/widgets/sliceviewer/test/test_sliceviewer_lineplots.py

```diff
@@ -35,8 +37,8 @@ def test_construction_adds_line_plots_to_axes(self, mock_gridspec):
         self.assertEqual(1, mock_gridspec.call_count)
         # use spaces at 0, 1 & 3 in grid
         gs.__getitem__.assert_has_calls((call(0), call(1), call(3)), any_order=True)
-        self.assertTrue('sharex' in fig.add_subplot.call_args_list[0].kwargs)
-        self.assertTrue('sharey' in fig.add_subplot.call_args_list[1].kwargs)
+        self.assertTrue('sharey' in fig.add_subplot.call_args_list[0][1] or 'sharey' in fig.add_subplot.call_args_list[1][1])
```

Comment:
Should this be an `or`? Is there a random aspect as to if `sharey` is in `[0][1]` or `[1][1]`?

Reply:
As far as I can tell it is not guaranteed to be in the correct order, and it reduces the potential brittleness of these tests. It doesn't matter which order they appear in the list, just that they are present.